docs/step-by-step/kubeapps-on-tkg/step-1.md

```diff
@@ -2,50 +2,11 @@
 
 The first step is to configure an OIDC provider ([VMware Cloud Service Portal (CSP)](https://console.cloud.vmware.com) login in this example) in the VMware Tanzu™ Kubernetes Grid™ (TKG) cluster and configure Pinniped to trust this provider for authenticating Kubernetes API calls.
 
-### Step 1.1: Install a Recent Version of Pinniped
+### Step 1.1: Configure an OIDC Provider
 
-> **NOTE**: Skip this section if [the cluster is already attached or will be attached to VMware Tanzu™ Mission Control™ (TMC)](https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-getstart/GUID-F0162E40-8D47-45D7-9EA1-83B64B380F5C.html). When a cluster is attached to TMC a recent Pinniped version compatible with Kubeapps is automatically installed in your cluster.
+TKG comes with Pinniped installed by default; you only need to configure an OIDC provider and then make Pinniped trust that OIDC provider. The OIDC provider is responsible for authenticating users in the TKG cluster and the Kubeapps dashboard. The steps below use the [VMware Cloud Services Platform (CSP)](https://console.cloud.vmware.com/) as an example; however, a similar process applies to any other OIDC-compliant provider, including Google Cloud, Azure Active Directory, Dex, Okta, and others.
 
-As per TKG v1.3.1, the built-in Pinniped version is relatively old (v0.4.1) and has some incompatibilities with the current Kubeapps releases. Specifically, Kubeapps requires Pinniped v0.6.0 or higher. However, Pinniped can be installed multiple times as a standalone product (not as a _TKG addon)_ in separate namespaces and using different Kubernetes API groups.
-
-> **NOTE**: Although users can install Pinniped in any namespace with any API group, due to current limitations in Kubeapps, the list of supported combination of namespace/API groups is limited to:
->
-> - `*.pinniped-dev` resources in the `pinniped-concierge` namespace.
-> - `*.pinniped.tmc.cloud.vmware.com` in the `vmware-system-tmc` namespace.
->   However, note that, even though the `tmc` name appears in the above namespace, it is only required for technical reasons. There is no requirement to use TMC when running Kubeapps on TKG.
```

Comment:
I think we should create a subsection or sth for TMC users, but I still have to figure out the best approach here.
Perhaps a section replacing the snippets with the TMC ones and just references to that section when appropriate in the rest of the sections. I don' know

Reply:
What happens currently if a user registers their TKG cluster with TMC? They get two pinniped's running? If so, yes, we can just note that and point to some other TMC-specific docs if they want to configure the TMC pinniped instead? But +1 to think about this a bit more and do it separately.

Reply:
Yes, another Pinniped instance is installed (in the `vmware-system-tmc` ns), but I don't know with is the preference here: whether using the TMC or the TKG Pinniped. I guess people using TMC would like to use this one (for authenticating to multiple clusters using their existing account).

So, I agree, let's just write a note for now (but let's revisit it if any TMC-related question pops up)

Reply:
The usual case i havr seen is actually still using the tkg provided pinniped and not the tmc one. I have yet to see a customer of tkg that uses the tmc pinniped